README.md

```diff
@@ -1,29 +1,30 @@
-<h1 align="center">Java Spring generator</h1>
-<p align="center">
-  <em>Use your AsyncAPI definition to generate java code to subscribe and publish messages</em>
-</p>
-<br><br>
+#Java Spring generator
+_Use your AsyncAPI definition to generate java code to subscribe and publish messages_
 
-# ATTENTION: This template is out of date. Please help us update it.
 
+# ATTENTION: This template is not complete. Please help us update it.
+Find more information in [Contribution](docs/contribution.md).
```

Comment:
there is not much content there and it is anyway not someting I would call `contribution`. You can place the content from `docs/contribution.md` directly in the readme, something like `missing features` section you see here https://github.com/asyncapi/asyncapi-react/#missing-features

Reply:
Makes sense. I moved it. Please see new commit